### ISSUE TITLE
Dynamic links have extra &nbsp;

### ISSUE THREAD

**Issue by ludwiczakpawel** [forem/forem#10980]
**Describe the bug**

All links in main navigation have some extra whitespace `&nbsp;` for some reason, making "More..." link looks like misaligned with rest.

**Expected behavior**

Links to not have that whitespace.

**Screenshots**

![image](https://user-images.githubusercontent.com/108287/96696633-47263400-138b-11eb-8a9d-0789aa308f4a.png)

**Comment by citizen428** [comment]
@ludwiczakpawel I think we shouldn't assign issues to specific people unless requested by them, this seems like everyone including external contributors could potentially fix this.

**Comment by citizen428** [comment]
FWIW, the `&nbsp;` is explicitly added in line 5 of `app/views/articles/_sidebar_nav_link.html.erb`:

```html
    <span class="crayons-icon crayons-icon--default" title="Listings">
      <%= link.icon.html_safe %>
    </span>
    &nbsp;<%= link.name %>

```

**Comment by ludwiczakpawel** [comment]
@citizen428 the reason for assigning specific person to this issue that way was because I was talking to that person at that exact moment when creating the issue. In addition, this person was actually on that specific area in the very moment of me reporting the issue and talking to this person. I believe my judgment, in this specific case, was pretty accurate.

> this seems like everyone including external contributors could potentially fix this

That's probably true statement for *almost every* issue.

**Comment by ludwiczakpawel** [comment]
Closing – resolved in #10981 . Cheers @Ridhwana 🙌
